files/build_templates/init_cfg.json.j2

```diff
@@ -61,7 +61,11 @@
             "state": "{{state}}",
             "has_timer" : {{has_timer | lower()}},
             "has_global_scope": {% if feature + '.service' in installer_services.split(' ') %}true{% else %}false{% endif %},
+{%- if feature in ["lldp"] %}
+            "has_per_asic_scope": {% raw %}"{% if not DEVICE_RUNTIME_METADATA['ETHERNET_PORTS_PRESENT'] or ('CHASSIS_METADATA' in DEVICE_RUNTIME_METADATA and DEVICE_RUNTIME_METADATA['CHASSIS_METADATA']['module_type'] in ['supervisor']) %}False{% else %}True{% endif %}"{% endraw %},
+{%- else %}
             "has_per_asic_scope": {% if feature + '@.service' in installer_services.split(' ') %}true{% else %}false{% endif %},
```

Comment:
has_per_asic_scope is changed from boolean_type to string, should we change here too ?

Reply:
@judyjoseph : thanks for the comment and catching this. Updated.